packages/snaps-controllers/src/interface/SnapInterfaceController.ts

```diff
@@ -474,6 +554,17 @@ export class SnapInterfaceController extends BaseController<
     return this.messagingSystem.call('SnapController:get', id);
   }
 
+  /**
+   * Whether if the snap owns the account.
+   *
+   * @param snapId - The snap id.
+   * @param account - The account.
+   * @returns True if the snap owns the account, otherwise false.
+   */
+  #snapOwnsAccount(snapId: SnapId, account: InternalAccount) {
+    return account.metadata.snap?.id === snapId;
```

Comment:
This doesn't use controller state and thus seems strange to include as a function on the class